recipes/domclust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: domclust 
+  version: "1.2.7l"
+
+about:
+  home: http://mbgd.genome.ad.jp/domclust/
+  license: 
+  summary: Effective tool for orthologous grouping in multiple genomes 
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: domclust.tgz 
+  url: "http://mbgd.genome.ad.jp/domclust/domclust.tgz"
```

Comment:
This does not look like a versioned URL.  If you do not trust this URL please mirror it here with https://github.com/galaxyproject/cargo-port